_includes/sections/instant-messenger.html

```diff
@@ -40,44 +67,199 @@ <h1 id="im" class="anchor"><a href="#im"><i class="fas fa-link anchor-icon"></i>
+  <a href=\"https://conversations.im/omemo/\">OMEMO</a> is the most popular XEP for E2EE (end-to-end encryption). Unlike Matrix, clients are only developed by the community and not by the foundation itself. We've linked some of the ones which we think are good choices below."
+  labels="warning:Warning inconsistent E2EE:VOIP and File transfers may not be end-to-end encrypted"
+  website="https://xmpp.org/"
+  forum=""
+  windows="https://gajim.org/"
+  mac="https://gajim.org/"
+  linux="https://gajim.org/"
+  fdroid="https://f-droid.org/en/packages/eu.siacs.conversations"
+  googleplay="https://play.google.com/store/apps/details?id=eu.siacs.conversations"
+  ios="https://monal.im"
+%}
+
+<div>
+    <h5>Worth Mentioning</h5>
+
+    <ul>
+        <li><a href="https://www.kontalk.org">Kontalk</a>, A community-driven instant messaging network. Supports E2EE (end-to-end encryption). Both client-to-server and server-to-server channels are fully encrypted.</li>
+        <li>Other <a href="https://matrix.org/clients">Matrix</a> clients may be less feature complete than Riot.im.</li>
+        <li>Other <a href="https://omemo.top">OMEMO</a> capable clients for XMPP.</li>
+    </ul>
+
+    <h3 id="im" class="anchor"><a href="#peer-to-peer"><i class="fas fa-link anchor-icon"></i></a> Peer to Peer</h3>
+
+    <p>These instant messengers connect directly to each other without an authoritative server in between. Peers (clients) usually negotiate connections through the use of some kind of <a href="https://en.wikipedia.org/wiki/Distributed_computing">distributed computing</a> network. Examples of this can include the <a href="https://en.wikipedia.org/wiki/Distributed_hash_table">DHT (Distributed hash table)</a>, or <a href="https://en.wikipedia.org/wiki/Ethereum">Ethereum</a>'s <a href="https://github.com/ethereum/wiki/wiki/Whisper">Whisper</a> protocol.</p>
+
+    <p>Once a peer has found the correct route to its partner, a direct connection to that contact is made. Another approach is proximity based networks, where a connection is established over WiFi or Bluetooth (eg Briar or the <a href="https://www.scuttlebutt.nz">Scuttlebutt</a> social networking protocol).</p>
+
+    <h4>Advantages</h4>
+    <ul>
+        <li>Minimal metadata exposed. Your <a href="https://en.wikipedia.org/wiki/IP_address">IP address</a> and that of the contacts you're communicating with may be visible if you do not use the software in conjunction with an anonymity network like <a href="https://www.torproject.org">Tor</a> or <a href="https://geti2p.net/">I2P</a>. Many countries have some form of mass surveillance and/or metadata retention.</li>
+        <li>Always on E2EE (end-to-end encryption), no ability to disable.</li>
+    </ul>
+
+    <h4>Disadvantages</h4>
+    <ul>
+        <li>Reduced feature set:</li>
+        <ul>
+            <li>No offline messaging (Tox, Jami).</li>
```

Comment:
```suggestion
            <li>No offline messaging (Tox, Jami), however messages may be stored locally to wait for the contact to be online simultaneously with your client</li>
```

I think something like this may make this seem like a bit less bad option.